addons/portal/static/src/xml/portal_security.xml

```diff
@@ -21,6 +21,20 @@
                 and complete, <strong>it will be the only way to
                 identify the key once created</strong>.
             </p>
+            <h3 class="fw-bold">
+                Give a duration for the key's validity
+            </h3>
+            <t t-foreach="duration_selection" t-as="duration" t-key="duration[0]">
+                <input type="radio" class="form-check-input ms-0 me-2" name="duration"
+                    t-att-id="`duration_${duration[0]}`"
+                    t-att-value="duration[0]"/>
+                <label class="form-check-label"
+                    t-att-for="`duration_${duration[0]}`" t-out="duration[1]"/>
+                <br/>
+            </t>
+            <p>
+                The key will be deleted once this period has elapsed.
+            </p>
```

Comment:
I was thinking that it looks a bit weird that there is only one option and it doesn't even auto select it
![image](https://github.com/user-attachments/assets/e8ed5cb4-87c4-4408-bae8-86d96fa0b2c1)

Reply:
I have changed the input from radio to a selection, which seems more appropriate (the first value is automatically selected).